mlrun/runtimes/local.py

```diff
@@ -239,10 +240,14 @@ def _run(self, runobj: RunObject, execution: MLClientCtx):
             return context.to_dict()
 
         else:
+            command = self.spec.command
+            if self.spec.mode and self.spec.mode in ["pass", "args"]:
```

Comment:
You added a validation for this in the beginning of `def run()`, so this will always be true, why conditioning again ?

Reply:
i want to do the format only in case of args & pass modes (not in default mode)